polling_stations/settings/constants/councils.py

```diff
@@ -2,5 +2,37 @@
 
 YVM_LA_URL = "https://www.yourvotematters.co.uk/_design/nested-content/results-page2/search-voting-locations-by-districtcode?queries_distcode_query="  # noqa
 BOUNDARIES_URL = (
-    "https://opendata.arcgis.com/datasets/e3634984fe1143da9fb31671627f5443_1.geojson"
+    "https://opendata.arcgis.com/datasets/593018bf59ab4699b66355bd33cd186d_1.geojson"
 )
+
+
+# The data we need won't all be
+# updated before May 2019
+# so this will help us to bodge it
+OLD_TO_NEW_MAP = {
+    # Fife
+    "S12000015": "S12000047",
+    # Perth & Kinross
+    "S12000024": "S12000048",
+    # Somerset West & Taunton
+    "E07000190": "SWT-tempid",
+    "E07000191": "SWT-tempid",
+    # Bournemouth, Christchurch & Poole
+    "E06000028": "BPC-tempid",
+    "E06000029": "BPC-tempid",
+    "E07000048": "BPC-tempid",
+    # Dorset
+    "E07000049": "DST-tempid",
+    "E07000050": "DST-tempid",
+    "E07000051": "DST-tempid",
+    "E07000052": "DST-tempid",
+    "E07000053": "DST-tempid",
+    # East Suffolk
+    "E07000205": "ESK-tempid",
+    "E07000206": "ESK-tempid",
+    # West Suffolk
+    "E07000201": "WSK-tempid",
+    "E07000204": "WSK-tempid",
+}
+
+NEW_COUNCILS = ["SWT-tempid", "BPC-tempid", "DST-tempid", "ESK-tempid", "WSK-tempid"]
```

Comment:
I've made up some fake codes here based on the GDS register codes, but I'm hoping we'll at least get a Code History DB release telling us what the new codes are going to be in the next couple of months :crossed-fingers: These are the same temp codes I've assigned these `OrganisationGeography` objects in Every Election.